### ISSUE TITLE
ncdf4 can't be loaded or installed on test-pecan

### ISSUE THREAD

**Issue by araiho** [PecanProject/pecan#1011]
This error occurs when you run a model or try to install ncdf4 on test-pecan:

Error in dyn.load(file, DLLpath = DLLpath, ...) : 
unable to load shared object '/usr/lib64/R/library/ncdf4/libs/ncdf4.so':
libhdf5.so.6: cannot open shared object file: No such file or directory
Error: package 'ncdf4' could not be loaded

**Comment by dlebauer** [comment]
Sounds similar to #998, #509, #426, #131, #71 and probably a few others

I think this is a FAQ and always a headache. I wish there were an easier solution ... but ... You need to either make sure the netcdf (and dependencies) module are loaded, or else specify the path when installing the package.

**Comment by dlebauer** [comment]
@araiho `module show netcdf` should help you find the paths for the netcdf libraries

**Comment by araiho** [comment]
@dlebauer  Unable to locate a modulefile for 'netcdf'

**Comment by ashiklom** [comment]
We don't have a module system, but we fixed it.

The trick was to add the following line to `/etc/rstudio/rserver.conf`:

```
rsession-ld-library-path=/usr/lib64/openmpi/lib:/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/usr/local/IT/bin
```

The key path in there is the `/usr/lib64/openmpi/lib`, which is where the required `libhdf5.so.6` is located. 

@dlebauer I've found a more general command to find locations is just `locate` or `find`. Even when modules are "loaded", I think their paths are sometimes still weird -- I've definitely had issues on our BU cluster.
